src/utils/parse_utils.ts

```diff
@@ -182,4 +182,11 @@ export const parseUtils = {
 
         return parsedConfig;
     },
+    parseAssetDatasStringFromQueryParam(field: string): string | string[] {
+        if (field.indexOf(',') !== 0) {
```

Comment:
Indexof returns `-1` if element is missing, so it should be `!== -1` rite? 

Personally I prefer `field.includes(',')` since it's a bit more readable
```suggestion
        if (field.indexOf(',') !== -1) {
```

Reply:
ugh lol yeah what was I thinking